.gitignore

```diff
@@ -40,3 +40,4 @@ coverage
 libpeerconnection.log
 /config/application.yml
 node_modules
+/vendor/bundle
```

Comment:
Yeah it's a fairly standard environment setup convention (see https://github.com/github/gitignore/blob/master/Rails.gitignore#L27) but nothing to do with this functionality so happy to pull it out, if you like?